client/pages/sections/protocols/steps.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
```

Comment:
If we're importing from `react-router` then it should be declared as a dependency in package.json.

Reply:
yes, yes we should. Didn't somebody write an eslint plugin exactly for this purpose  🤔

Reply:
It doesn't understand `import` because someone is too lazy to update it.